src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -231,7 +266,139 @@ export const handleEncryptedSubmission: RequestHandler = async (
       myinfoResult.value
   }
 
-  return next()
+  // Encrypt Verified SPCP Fields
+  if (form.authType === AuthType.SP || form.authType === AuthType.CP) {
+    const encryptVerifiedContentResult = VerifiedContentFactory.getVerifiedContent(
+      { type: form.authType, data: res.locals },
+    ).andThen((verifiedContent) =>
+      VerifiedContentFactory.encryptVerifiedContent({
+        verifiedContent,
+        formPublicKey: form.publicKey,
+      }),
+    )
+
+    if (encryptVerifiedContentResult.isErr()) {
+      const { error } = encryptVerifiedContentResult
+      logger.error({
+        message: 'Unable to encrypt verified content',
+        meta: logMeta,
+        error,
+      })
+
+      // Passthrough if feature is not activated.
+      if (error instanceof MissingFeatureError) {
+        return next()
+      }
+
+      return res
+        .status(StatusCodes.BAD_REQUEST)
+        .json({ message: 'Invalid data was found. Please submit again.' })
+    }
+  }
+
+  // Verify Structure of Encrypted Response
+  // Step 1: Add req.body.encryptedContent to req.formData
+  // eslint-disable-next-line @typescript-eslint/no-extra-semi
+  ;(req as WithFormData<typeof req>).formData = req.body.encryptedContent
+  // Step 2: Add req.body.attachments to req.attachmentData
+  ;(req as WithAttachmentsData<typeof req>).attachmentData =
+    req.body.attachments || {}
```

Comment:
previously we had to tack on `attachmentData` to `req` because of long-range dependencies across multiple middleware, but i think we can dispense with this pattern now :)

Reply:
Yeah, realized this after I finished haha. Removing...